generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -6,6 +6,10 @@ import { TranslateService } from 'ng2-translate';
 <%_ } _%>
 import { AlertService } from 'ng-jhipster';
 
+<%_ if (websocket === 'spring-websocket') { _%>
+import { WindowRef } from './tracker/window.service';
```

Comment:
Ah sorry I missed this. Window service is common right. That means we should be using this instead of direct window reference everywhere. That means we can actually move the window service into ng-jhipster lib and use it here. what do you think? I think its used for csrf-service as well